deltachat-ffi/src/lib.rs

```diff
@@ -144,6 +144,34 @@ pub unsafe extern "C" fn dc_context_is_open(context: *mut dc_context_t) -> libc:
     block_on(ctx.is_open()) as libc::c_int
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn dc_context_is_encrypted(context: *mut dc_context_t) -> libc::c_int {
+    if context.is_null() {
+        eprintln!("ignoring careless call to dc_context_is_encrypted()");
+        return 0;
+    }
+
+    let ctx = &*context;
+    block_on(ctx.is_encrypted()) as libc::c_int
+}
+
+#[no_mangle]
+pub unsafe extern "C" fn dc_context_change_passphrase(
+    context: *mut dc_context_t,
+    passphrase: *const libc::c_char,
+) -> libc::c_int {
+    if context.is_null() {
+        eprintln!("ignoring careless call to dc_context_change_passphrase()");
+        return 0;
+    }
+
+    let ctx = &*context;
+    let passphrase = to_string_lossy(passphrase);
```

Comment:
I think I'd rather error out if the passphrase isn't UTF-8 instead of setting it to something else, even if that something else is kind of predictible.

Reply:
do you mind addressing this?  otherwise lgtm